app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,70 @@
+!!!
+%html
+  %head
+    %meta{content: "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}
+    %title ProDobro
+    = csrf_meta_tags
+    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload'
+    = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
+    %link{href: "/images/hand.png", rel: "shortcut icon", type: "image/x-icon"}
```

Comment:
Please use [asset tag helpers](http://guides.rubyonrails.org/layouts_and_rendering.html#asset-tag-helpers) for assets